sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -513,10 +512,14 @@ kernel_pid_t gnrc_ipv6_netif_find_by_prefix(ipv6_addr_t **out, const ipv6_addr_t
         mutex_unlock(&(ipv6_ifs[i].mutex));
     }
 
+    if (out != NULL) {
+        *out = found;
```

Comment:
Since code size comes before DEBUGING needs, I would prefer to `assert()` that `out != NULL` at the beginning of the function (and document that precondition). If `out` is `NULL` this might point to a bug somewhere else and we should fix it rather than the symptoms.